### PR TITLE
GraphQL: Add fixed product tax attributes

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) adds attributes that support fixed product taxes. It also includes miscellaneous updates for minor schema changes from Community Engineering PRs and linting.

## Affected DevDocs pages

- guides/v2.3/graphql/product/product-interface.md
- guides/v2.3/graphql/queries/store-config.md

whatsnew
Added attributes that support fixed product taxes to the [ProductInterface](https://devdocs.magento.com/guides/v2.3/graphql/product/product-interface.html) and [`storeConfig`](https://devdocs.magento.com/guides/v2.3/graphql/queries/store-config.html) query.

### CODE REVIEW

File: guides/v2.3/graphql/queries/store-config.md [hunk 1]
```diff
@@ -177,6 +177,36 @@ The following query returns information about the store's catalog configuration.
 }
 ```
 
+### Query a store's fixed product tax configuration
+
+The following query returns enumeration values that indicate the store's fixed product tax configuration,
```
Comment (jeff-matthews) [inline guides/v2.3/graphql/queries/store-config.md]:
```suggestion
The following query returns enumeration values that indicate the store's fixed product tax configuration.
```

File: guides/v2.3/graphql/product/product-interface.md [hunk 2]
```diff
@@ -98,6 +98,7 @@ Attribute |  Data Type | Description
 --- | --- | ---
 `discount` | ProductDiscount | The amount of the discount applied to the product. It represents the difference between the `final_price` and `regular_price`
 `final_price`| Money! | The price of the product after applying discounts
+`fixed_product_taxes` | [[FixedProductTax](#FixedProductTax)] | An array of fixed product taxes applied to a product price
```
Comment (cpartica) [inline guides/v2.3/graphql/product/product-interface.md]: it's not necessarily applied. we might display then without affecting final price

File: guides/v2.3/graphql/queries/store-config.md [hunk 3]
```diff
@@ -245,14 +275,35 @@ Attribute |  Data Type | Description | Example
...
+--- | --- | ---
+`category_fixed_product_tax_display_setting` | FixedProductTaxDisplaySettings | Corresponds to the **Display Prices In Product Lists** field. It indicates how FPT information is displayed on category pages
+`product_fixed_product_tax_display_setting` | FixedProductTaxDisplaySettings | Corresponds to the **Display Prices On Product View Page** field. It indicates how FPT information is displayed on product pages
+`sales_fixed_product_tax_display_setting` | FixedProductTaxDisplaySettings | Corresponds to the **Display Prices In Sales Modules** field. It indicates how FPT information is displayed on cart, checkout, and order pages
+
+The `FixedProductTaxDisplaySettings` data type is an enumeration that describes whether displayed prices include fixed product taxes and whether Magento separately displays detailed information about the FPTs.
+
+Value | Description
+--- | ---
+EXCLUDE_FPT_AND_INCLUDE_WITH_DETAILS | The displayed price does not include the FPT amount. The values of `ProductPrice.fixed_product_taxes` and the price including the FPT are displayed separately. This value corresponds to **Excluding FPT, Including FPT description and final price**
```
Comment (cpartica) [inline guides/v2.3/graphql/queries/store-config.md]: could we add these in the order as they are in the admin?

Comment (cpartica) [inline guides/v2.3/graphql/queries/store-config.md]: it's not very clear that you have to display Price excluding, and also including, so there's 2 prices + the the individual FPT s

Comment (keharper) [reply guides/v2.3/graphql/queries/store-config.md]: They're in alphabetical order, like all the other attributes.

**Pull request merged**